src/WebApi/HealthChecks/EntityFrameworkDbContextHealthCheck/DbHealthCheckResult.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SSW.CleanArchitecture.WebApi.HealthChecks.EntityFrameworkDbContextHealthCheck;
+
+public sealed class DbHealthCheckResult
```

Comment:
Consider adding one or more constructors to force properties to be initialized.  Properties could then have private setters.

Reply:
done